src/Service/CsvParser.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service;
+
+use App\Factory\ProductFactory;
+use Symfony\Component\Finder\Finder;
+
+class CsvParser
+{
+    private string $productsFileDir;
+
+    private string $productsFileName;
+
+    private const FILE_EXT = '.csv';
```

Comment:
constans should be defined at the beginning of the file, before all the properties